src/functions/onSetUsername.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Perhaps add this rule to eslintrc if this exists on all functions.

Could you also add 
```
    "prettier/prettier": [
      "error",
      {
        "endOfLine": "auto"
      }
    ]
```
Prettier giving linebreak errors on Windows.

Reply:
It's strange that my editor gives me this warning only for the `aws-lambda` package, saying that it cannot find it.

Reply:
kk, im going to add these rules